THIRD-PARTY-NOTICES.md

```diff
@@ -0,0 +1,370 @@
+
+MsQuc uses third-party libraries or other resources that may be
+distributed under licenses different than the MsQuic software.
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention. Post an issue or email us:
+
+           quicdev@microsoft.com
```

Comment:
Let's just have them post an issue or discussion. No email (this is an internal address).